src/lidartag.cpp

```diff
@@ -2482,7 +2519,7 @@ bool LidarTag::transformSplitEdges(ClusterFamily_t & cluster)
       "Actual Points: " << cluster.data.size() + cluster.edge_points.size());
   }
 
-  int num_edge_points = 3;
+  int num_edge_points = 2;
```

Comment:
Why did you reduce this value?

Reply:
It was an experiment regarding the extreme cases of the input pointcloud, but I better revert it to the previous value. Thanks ! :bow: